database/src/main/postgres/flows/V1.9.1__get_flow_checkpoints_v2.sql

```diff
@@ -66,53 +70,60 @@ $$
 --      measured_columns       - measure columns associated with a given checkpoint
 --      measurement_value      - measurement details associated with a given checkpoint
 --      checkpoint_time        - time
```

Comment:
--      checkpoint_start_time   - Time of the checkpoint
--      checkpoint_end_time     - End time of the checkpoint computation